README.md

```diff
@@ -11,32 +11,27 @@
 [//]: # (</p>)
 
 ## 👵 Welcome Home!
-PyTorch lets you send a model or tensor `.to(device)`, so
-why can't you do `my_fn.to('a_gcp_a100')` or `my_table.to('parquet_in_s3')`?
-Runhouse allows just that: send code and data to any of your compute or
-data infra (with your own cloud creds), all in Python, and continue to interact with
-them normally from your existing code and environment. Think of it as an **expansion pack to your Python
-interpreter** that lets it take detours to remote machines or manipulate remote
-data. It wraps industry-standard tooling like Ray, gRPC, and the Cloud SDKs (boto, gsutil, etc. via [SkyPilot](https://github.com/skypilot-org/skypilot/))
-to give you production-quality features like queuing, distributed, async, logging,
-low latency, auto-launching, and auto-termination out of the box.
-
-Runhouse is for ML Researchers, Engineers, and Data Scientists who are tired of:
- - 🚜 manually shuttling code and data around between their local machine, remote instances, and cloud storage,
- - 🐜 debugging over ssh and notebook tunnels,
- - 🧑‍🔧 translating their code into a pipeline DSL just to use multiple hardware types,
- - 🪦 debugging in an orchestrator,
- - 👩‍✈️ missing out on fancy LLM IDE features,
- - 🕵️ and struggling to find or reproduce their teammates' code and data artifacts.
-
-By way of a visual,
+
+ML infra is complicated and fragmented. Runhouse helps unify your ML stack, reducing the need for
+any code translation or data migration, and lets you save, share, and manage your evolving code and
+data artifacts, all in plain Python.
```

Comment:
ML infra is complicated and fragmented, but Runhouse provides a collaborative and unified interface for sharing and managing our ML stack. We bridge the gap between:
- research and production
- local and remote